docs/wymeditor_development/making_releases.rst

```diff
@@ -184,9 +187,11 @@ Ship it!
 Prepare for the next release
 ============================
 
-#. Create a new version entry in the change–log.
-#. Bump the version string and add the build
-   metadata string, ``+dev``, at the end.
+Since you're not sure what the next version is going to be,
+we'll call it ``next``.
+
+#. Create a new version entry in the change–log for ``next``.
+#. Bump the version string to ``next``.
```

Comment:
The new version entry in the changelog is created for `next`.

The version is bumped to `next` without `+dev` suffix.